### DISCUSSION TITLE
camGame object camera

### DISCUSSION THREAD

**Discussion by Refenger** [Leather128/LeathersFunkinEngine#304] [Q&A] [answered]
sry i found my answer but cant remove discussion

**Comment by karstenx10** [comment]
answer it now

**Comment by Refenger** [comment, marked as answer]
i see everybody use set object camera to hud but DID YOU KNOW 😱 that you can set its camera to the game camera by simply putting camGame instead of hud???? that's the answer, good luck

**Comment by karstenx10** [comment]
mark this as the answer now it should work.
